README.md

```diff
@@ -155,6 +155,19 @@ To set up the mediator storage (MongoDB):
 - `MONGODB_PASSWORD` - is the password used by the Mediator service to connect to the database.
 - `MONGODB_DB_NAME` - is the name of the database used by the Mediator.
 
+#### Mediator storage 
+- The `messages` collection contains two types of messages: `Mediator` and `User`.
+1. **Mediator Messages**:
+    - These messages received by mediator for any interactions with the mediator.
+    - Examples include messages for setting up mediation, requesting mediation, or picking up messages from the mediator.
+    - These messages stored in collection can be used for debugging purpose mediator functionality and interactions with the mediator. Hence they can be deleted after a period of time.
+    - This message type `Mediator` can be setup to have a configurable Time-To-Live (TTL) value, after which they can expire.
+    - This is how the TTL can be configured for the collection messages [initdb.js](initdb.js)
+2. **User Messages**:
+    - These are the actual messages e.g Forward message that are being mediated (stored) and delivered to users they have the the type `User`.
```

Comment:
I think this part is not clear enough.
Ex `Forward message` is a `Mediator` Message send to the mediator. The messaging inside of the Forward message is the one to the `User`.

Reply:
Let me fix this